Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -60,6 +62,20 @@ static CursorType cursorTypeForElement(Element& element)
     return cursorType;
 }
 
+static bool skipNonFullscreenElements(const RenderObject& regionRenderer)
```

Comment:
This function name reads like an action ("please skip") but really is a query ("should skip?")